### ISSUE TITLE
Make the Enterprise sign in flow more prominent

### ISSUE THREAD

**Issue by chenkc805** [sourcegraph/jetbrains#287]
### Description:

TLDR - we need to make it more obvious on our signup flow to Enterprise users for where they go to signup on their Enterprise instance. 

Spec here: https://docs.google.com/document/d/1zdrQhLUTAx3mhV8qIUbMFXK5bgb-DGm6jpnKh96KRn0/edit#heading=h.fj8cdyv2vg9r

Figma here: https://www.figma.com/file/X241xtVLpGmAexR08FW1RG?node-id=11:2&mode=design#665409806
![Screenshot 2024-01-11 at 11 49 33 AM](https://github.com/sourcegraph/jetbrains/assets/3654603/1dec4735-72da-45fe-8596-ac950563273a)

Please assign me the PR to review before this goes out. 

### Design Tasks:

```[tasklist]
### Design Tasks
- [ ] Add a draft title or issue reference here
```

### Engineering Tasks:

```[tasklist]
### Engineering Tasks
- [ ] Add a draft title or issue reference here
```

### Related Docs:

- [PRD: Your-Title]()
